grafana/Dockerfile

```diff
@@ -1,13 +1,67 @@
 ARG GRAFANA_VERSION
 
-FROM grafana/grafana:${GRAFANA_VERSION} as grafana-builder
+# Fetch the sources
+FROM alpine:3.16 as fetcher
+ARG GRAFANA_VERSION
```

Comment:
why we have it twice? Can't you remove line 1 then?

Reply:
fixed.